test/evaluation/evaluationClient.spec.ts

```diff
@@ -68,21 +75,21 @@ describe('evaluate', () => {
 
   it('should return ok if uri in vp matches at least one of input_descriptor\'s uris', function () {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
     vpSimple.holder = HOLDER_DID;
-    evaluationClient.evaluate(presentationDefinition, new VP(vpSimple));
+    evaluationClient.evaluate(presentationDefinition, vpSimple);
     const errorResults = evaluationClient.results.filter(result => result.status === Status.ERROR);
     expect(errorResults.length).toEqual(0);
   });
 
   it('should return error if uri in verifiableCredential doesn\'t match', function () {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.